docs/src/main/asciidoc/intro.adoc

```diff
@@ -164,6 +164,34 @@ include::https://raw.githubusercontent.com/spring-cloud-samples/sleuth-documenta
 NOTE: If you're using a custom `logback-spring.xml` then you have to pass the `spring.application.name` in
 `bootstrap` instead of `application` property file. Otherwise your custom logback file won't read the property properly.
 
+===== Propagating Span Context
+
+The span context is the state that must get propagated to any child Spans across process boundaries.
```

Comment:
here and above, you can clarify like this. "...across process boundaries. For example, the trace and span IDs are a required part of the span context. Baggage is an optional part."